Added: alt attribute to few images (#19404)
mdn/content: files/en-us/learn/accessibility/wai-aria_basics/index.md

If you go to VoiceOver's landmarks menu (accessed using VoiceOver key + U and then using the cursor keys to cycle through the menu choices), you'll see that most of the elements are nicely listed so they can be accessed quickly.

> ![four landmarks namely banner, navigation, main, complementary are listed by Mac's VoiceOver menu for quick accessibility](landmarks-list.png)

Reviewer: ```suggestion
![Mac's VoiceOver menu for quick accessibility. Landmarks header and landmarks list including banner, navigation, main, and complementary.](landmarks-list.png)
```
